trieste/acquisition/function.py

```diff
@@ -697,6 +697,115 @@ def ehvi_based_on_partitioned_cell(
     return acquisition
 
 
+class BatchMonteCarloExpectedHypervolumeImprovement(SingleModelAcquisitionBuilder):
+    """
+    Builder for the batch expected hypervolume improvement acquisition function.
+    The implementation of the acquisition function largely
+    follows :cite:`daulton2020differentiable`
+    """
+
+    def __init__(self, sample_size: int = 512, *, jitter: float = DEFAULTS.JITTER):
```

Comment:
what does the * in here mean?

Reply:
this follows the convention of [`BatchMonteCarloExpectedImprovement`](https://github.com/secondmind-labs/trieste/blob/28243c5fd3ff3bec89ad3621fb2a5a2c0b34c6bd/trieste/acquisition/function.py#L711), where i assume the intention of `*` is used for forcing `jitter` must be specified only in kwargs manner,  (not something like `BatchMonteCarloExpectedHypervolumeImprovement(1000, 1e-10)`), CMIIW.